src/CoreWCF.Primitives/src/CoreWCF/Description/MessagePartDescription.cs

```diff
@@ -48,7 +48,18 @@ internal MessagePartDescription(MessagePartDescription other)
             //this.uniquePartName = other.uniquePartName;
         }
 
-        internal virtual MessagePartDescription Clone()
+        internal MessagePartDescription(MessagePartDescription other, string name, string ns) : this(other)
+        {
+            XmlName = new XmlName(name, true /*isEncoded*/);
+            Namespace = ns;
+        }
+
+        public virtual MessagePartDescription Clone()
+        {
+            return new MessagePartDescription(this);
+        }
+
+        public virtual MessagePartDescription Clone(string name, string ns)
         {
```

Comment:
This is a new method not on .NET Framework. I'm not seeing the purpose of this overload of Close as you don't use the passed in parameters.

Reply:
This has been addressed.